cwms-data-api/src/test/resources/cwms/cda/data/dto/project2/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "location" : {
+    "office-id" : "SPK",
+    "name" : "Project Id",
+    "timezone-name": "UTC"
+  },
+  "federal-cost" : 100.0,
+  "non-federal-cost" : 50.0,
+  "cost-year" : 1717199914902,
```

Comment:
they've asked for dates as epoch millis instead of an iso string?

Reply:
CDA will deserialize epoch milli or ISO String into Instant.  CDA serializes Instant to epoch milli - thats turned on in the JSONv2 Formatter,  I think that was added recently with the text-timeseries and binary-timeseries work so its probably not locked in if you wanted to suggest changing the default.  I suspect there is a way to customize individual properties but U haven't tried.  Epoch millis is compact but not very readable.